recipes/pyopenbt/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyopenbt" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyopenbt-{{ version }}.tar.gz
+  sha256: 111caf73073c397c531b68b2ded50c5f4a2b751b14a3dd4c4c546a9f54318362
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.7
```

Comment:
Is there a reason the minimum python version in the `run` section is different than the one in the `host` section?

Reply:
now there isn't and that is an oversight. I will fix it now.

Reply:
Change sent